instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/ConsentGrantedSrTest.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sdk.integration.sessionreplay
+
+import androidx.test.platform.app.InstrumentationRegistry
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.sdk.integration.RuntimeConfig
+import com.datadog.android.sdk.integration.log.LogsTest
+import com.datadog.android.sdk.rules.SessionReplayTestRule
+import com.datadog.tools.unit.ConditionWatcher
+import org.junit.Rule
+import org.junit.Test
+
+internal class ConsentGrantedSrTest : SrSnapshotTest() {
+
+    @get:Rule
+    val rule = SessionReplayTestRule(
+        SessionReplayPlaygroundActivity::class.java,
+        trackingConsent = TrackingConsent.GRANTED,
+        keepRequests = true
+    )
+
+    @Test
+    fun verifySessionFirstSnapshot() {
+        // Wait to make sure all batches are consumed
+        InstrumentationRegistry.getInstrumentation().waitForIdleSync()
+
+        ConditionWatcher {
+            // verify the captured log events into the MockedWebServer
+            verifyExpectedSrData(
+                rule.getRequests(RuntimeConfig.sessionReplayEndpointUrl),
+                runInstrumentationScenario(rule)
+            )
+            true
+        }.doWait(timeoutMs = LogsTest.INITIAL_WAIT_MS)
```

Comment:
probably should be `SrTest.WAIT_MS` (here and in other tests)

Reply:
ah yes, definitely